tests/unit/algorithms/action/adapters/mmaction/models/backbones/test_action_register_backbone.py

```diff
@@ -16,4 +16,5 @@ def test_register_action_backbones() -> None:
     Since this function is called while initialization, X3D should be in mmdet backbone registry
     """
 
-    assert "X3D" in MMDET_BACKBONES
```

Comment:
Do this cause error? Since we use X3D as a action detection model backbone, it should be in MMDET_BACKBONES

Reply:
No error was caused, umm.. X3D is also in mmaction backbones. Do you want to leave as-is?

Reply:
Updated. Please review again.